src/odemis/gui/cont/fastem_project_tree.py

```
@@ -608,7 +625,8 @@ def _remove_tree_node(self, node: FastEMTreeNode):
         item = self._find_item_by_node(node)
         if item:
             self.Delete(item)
-            del self.node_window[node]
+            for node in node.get_all_nodes():
```

Ruff (0.15.15)

[warning] Loop control variable `node` overrides iterable it iterates

(B020)